UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -124,7 +124,7 @@
 				independent organizations.</p>
 
 			<p>This document proposes a shared framework for presenting publication accessibility metadata in a
```

Comment:
I realize I'm going off-topic here, but this last paragraph is the only one I'd consider the abstract. The preceding ones seem like they should form an introduction. The text from "It is useful to point out..." probably also doesn't belong in an abstract.

Again, you might want to do this in a separate pull request.